articles/virtual-desktop/store-fslogix-profile.md

```diff
@@ -11,11 +11,11 @@ ms.author: helohr
 ---
 # Storage options for FSLogix profile containers in Windows Virtual Desktop
 
-Azure offers multiple storage solutions that you can use to store your FSLogix profile container. This article compares storage solutions Azure Storage offers for Windows Virtual Desktop FSLogix profile container user profiles.
+Azure offers multiple storage solutions that you can use to store your FSLogix profile container. This article compares storage solutions that Azure offers for Windows Virtual Desktop FSLogix user profile container.
 
 Windows Virtual Desktop offers FSLogix profile containers as the recommended user profile solution. FSLogix is designed to roam profiles in remote computing environments, such as Windows Virtual Desktop. At sign-in, this container is dynamically attached to the computing environment using a natively supported Virtual Hard Disk (VHD) and a Hyper-V Virtual Hard Disk (VHDX). The user profile is immediately available and appears in the system exactly like a native user profile.
 
-The following tables compare the storage solutions Azure Storage offers for Windows Virtual Desktop FSLogix profile container user profiles.
+This following tables compare the storage solutions Azure Storage offers for Windows Virtual Desktop FSLogix profile container user profiles.
```

Comment:
"The following tables" is correct. "This" is used for one singular table, but there are two of them. (They were originally one single table, but they were split in two to prevent the table from getting too long.)